File: day060/homework/code/codewars1.py
# https://www.codewars.com/kata/55f2b110f61eb01779000053/train/python

def get_sum(a, b):
    if a == b:
        return a
    low = min(a, b)
    high = max(a, b)
    n = high - low + 1
    total = n * (low + high) // 2
    
    return total

File: day068/homework/code/codewars1.js
// https://www.codewars.com/kata/555eded1ad94b00403000071/train/python
def series_sum(n):
    if n == 0:
        return "0.00"
    
    total = 0.0
    for i in range(n):
        denominator = 1 + 3 * i
        total += 1 / denominator
    
    return f"{total:.2f}"